ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-stable-4.13-loaded-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,86 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-loaded-upgrade-412to413
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  intermediate:
+    release:
+      channel: candidate
+      version: "4.12"
+  latest:
+    release:
+      channel: candidate
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-multiaz-6nodes-stage-loaded-upgrade412
```

Comment:
@liqcui Also, since this test has nothing to do with 4.13, can we move this test to a separate file called `4.12 intermediate loaded upgrade` or something like that?
I am thinking that as we add more and more tests for different versions, it will get confusing and keeping the tests clean and separate will be more helpful. LMKWYT.

Reply:
Separated two files, thanks!